### PR TITLE
Add JS Question on Valid Variable Names

### PR DESCRIPTION
**Pull request by khairalanam**
## Summary of changes

This PR adds a question to the JavaScritp Quiz on valid variable names

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -2073,6 +2073,17 @@ const javascriptQuiz = [
...
+  {
+    Question:
+      "Which of the following is not a valid way to declare a variable in JavaScript?",
+    Answer: "let 1z = 10;",
+    Distractor1: "var x;",
+    Distractor2: "const y = 5;",
+    Distractor3: "let name = 'John'",
+    Explanation:
+      "Variable names cannot start with a number or any special character.",
+    Link: "https://www.w3schools.com/js/js_variables.asp"
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2086]: we only accept links to freeCodeCamp resources or MDN
please change this

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/javascript-quiz.ts (hunk 1 above)
Comment (khairalanam) [reply src/data/javascript-quiz.ts:2086]: okay sure

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**